app/views/hyrax/dashboard/collections/_form_for_select_collection.html.erb

```diff
@@ -52,9 +52,21 @@
           </div>
         <% else %>
           <div class="modal-body">
-            Please login to access your collections.
+            <p>Please login or create an account to save this work to a collection:</p>
+            
+            <%= button_tag "<i class='fa fa-lock' aria-hidden='true'></i> #{t('hyrax.dashboard.my.action.log_in')}".html_safe,
+                class: 'btn btn-primary btn-outline login-button',
+                data: { toggle: "modal", dismiss: "modal", target: "#user-login-modal",
+                tabindex: -1 } %>
+      
+            <%= link_to t('hyrax.dashboard.my.action.sign_up'), main_app.new_user_registration_path,
+                class: 'btn btn-primary login-button',
+                tabindex: -1 %>
```

Comment:
This is actually an already odd a11y fail going on with the site. Not sure who decided to take the sign-up (and login) button out of tab order, but those need to be tabbable.